public/style/main.css

```diff
@@ -0,0 +1,245 @@
+body{
+    font-family: 'Nanum Gothic', sans-serif;
+}
+
+.head-title {
+    color: #6b510a
+}
+
+
+.outer-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+}
+
+.description{
+    font-size:2vw;
+    width:80%;
+    text-align:center;
+}
+
+.container-1 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.container-2,.container-3,.container-4,.container-5{
```

Comment:
Is there a more descriptive name for these classes? I mean something that refers to what it is, for example: `header`, `button`, `dropdown-expanded`, `menu-wrapper` or something? Just to make them more individual.

Reply:
Actually they are the container screens. So this!